common/src/View/SetBrushFaceAttributesTool.cpp

```diff
@@ -38,6 +38,8 @@
 
 namespace TrenchBroom {
     namespace View {
+        static const std::string TransferFaceAttributesTransactionName = "transferFaceAttributes";
```

Comment:
I think this name is user visible in the undo menu entry, or at least it's supposed to be.

Reply:
Yeah, it is. I'll change it to "Transfer Face Attributes"